components/case-study/Results.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div class="container py-5">
+    <div class="heading d-flex flex-column">
+      <span class="section__subtext">Results</span>
+      <h2 class="section__heading">
+        Transforming Lead Generation with Formester Web Forms
+      </h2>
+      <p class="section__description">
+        The strategic adoption of Formester as your lead generation form builder
+        is capable of yielding impressive outcomes, revolutionizing your
+        approach to lead generation and conversion
+      </p>
+    </div>
+
+    <div class="templates mt-md-5">
+      <div v-for="(feature, idx) in features" :key="idx">
+        <div
+          class="card d-flex align-items-center justify-items-center text-center align-items-md-start text-md-start"
+        >
+          <nuxt-img
+            :src="`case-study-1/results/${convertToImageName(
+              feature.title
+            )}.svg`"
+            class="img-fluid"
+            loading="lazy"
+          />
+          <h4 class="sub__section-heading mt-4">{{ feature.title }}</h4>
+          <p class="subheading__text">{{ feature.description }}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      features: [
+        {
```

Comment:
`features` is an incorrect name. If they are results they should be named `results`

Reply:
@piyushsinghania let it me for now. I have seen that this type of section is already used on many different pages. So, we would need to break it component wise and refactor other pages.